### ISSUE TITLE
Accessibility errors in search form

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#31]
The search form is triggering multiple errors in SiteImprove for all websites using the template. Here's a summary of the problems: 
- The `<select>` field (for mobile) has no label. Solution: Add aria-label="Search scope" to `<select>`
- The submit button, radio buttons, and select field are all outside of the `<form>` element. They should be inside the `<form>` element. 
- An outermost `<div>` containing all  search widget content (div.uw-search-wrapper) should include role="search"

**Comment by nwinkelbauer** [comment]
Solved with commit 910daec and PR #45
